modules/SDL.lua

```diff
@@ -573,6 +573,34 @@ function SDL.AppInfo.clean()
   deleteFile(SDL.AppInfo.file())
 end
 
+SDL.HMICapCache = {}
+
+function SDL.HMICapCache.file()
+  return getFilePath(SDL.INI.get("HMICapabilitiesCacheFile"), SDL.INI.get("AppStorageFolder"))
+end
+
+function SDL.HMICapCache.get()
+  local content = getFileContent(SDL.HMICapCache.file())
+  return content and json.decode(content) or nil
```

Comment:
@aderiabin Are json.decode/encode functions always successful? Do we need to check the result for errors?

Reply:
@AByzhynar Yes, result should be checked, and it will be done in separate PR for all functions of SDL module.